backend/src/main/java/de/neuefische/backend/service/MongoUserDetailsService.java

```diff
@@ -2,26 +2,36 @@
 
 import de.neuefische.backend.model.UserMongo;
 import de.neuefische.backend.repo.MongoUserRepository;
-import org.springframework.security.core.userdetails.User;
-import org.springframework.security.core.userdetails.UserDetails;
+import org.apache.juli.logging.Log;
+import org.apache.juli.logging.LogFactory;
 import org.springframework.security.core.userdetails.UserDetailsService;
 import org.springframework.security.core.userdetails.UsernameNotFoundException;
 import org.springframework.stereotype.Service;
 
+import java.security.Principal;
+
 @Service
 public class MongoUserDetailsService implements UserDetailsService {
     public static final String AUTHORITY_API_READWRITE = "API_READWRITE";
+    private static final Log LOG = LogFactory.getLog(MongoUserDetailsService.class);
     private final MongoUserRepository repository;
 
     public MongoUserDetailsService(MongoUserRepository repository){
         this.repository = repository;
     }
 
     @Override
-    public UserDetails loadUserByUsername(String username) throws UsernameNotFoundException {
+    public UserMongo loadUserByUsername(String username) throws UsernameNotFoundException {
```

Comment:
Imho it is confusing that the object es of type UserMongo and the service is called MongoUserDetailsService, be consistent with your names

Reply:
Yeah, you're right, could be confusing. I will change that